drivers/can/stm32_ll_can.c

```diff
@@ -0,0 +1,725 @@
+/*
+ * Copyright (c) 2018 Alexander Wachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <clock_control/stm32_clock_control.h>
+#include <clock_control.h>
+#include <misc/util.h>
+#include <string.h>
+#include <kernel.h>
+#include <board.h>
+#include <errno.h>
+#include <can.h>
+#include "stm32_ll_can.h"
+
+#define SYS_LOG_LEVEL CONFIG_SYS_LOG_CAN_LEVEL
+#include <logging/sys_log.h>
+
+static const struct can_driver_api can_api_funcs = {
+	.configure = can_stm32_runtime_configure,
+	.send = can_stm32_send,
+	.attach_msgq = can_stm32_attach_msgq,
+	.attach_isr = can_stm32_attach_isr,
+	.detach = can_stm32_detach
+};
+
+static void signal_tx_complete(struct can_mailbox *mb)
+{
+	if (mb->tx_callback) {
+		mb->tx_callback(mb->error_flags);
+	} else   {
+		k_sem_give(&mb->tx_int_sem);
+	}
+}
+
+static inline void can_get_msg_fifo(CAN_FIFOMailBox_TypeDef *mbox,
+				    struct can_msg *msg)
+{
+	if (mbox->RIR & CAN_RI0R_IDE) {
+		msg->ext_id = mbox->RIR >> CAN_RI0R_EXID_Pos;
+		msg->id_type = can_extended_identifier;
+	} else   {
```

Comment:
remove extra spaces after else

Reply:
I used uncrustify for this file and it changed it from if \n {} \n else.. to this. I will change that.